lib/services/permissionvalidation.service.js

```diff
@@ -25,7 +26,12 @@ async function checkPermission(authToken, requiredPermissions, requestedsource)
   const configuration = config.getConfig();
   const { sources } = configuration;
   const source = requestedsource || configuration.source;
-
+  if (configuration.disabled) {
+    if (config.getConfig().debug) {
```

Comment:
it shouldn't just log this in debug mode, it should log it always, and the text should be very clear: "Authz protection is disabled, all actions by all users will be allowed."